frontend/src/reducers/auth.ts

```diff
@@ -59,11 +60,9 @@ export const logoutUserReducer = (state = {}, action: LogoutUserAction) => {
   }
 };
 
-const SessionData: AuthenticationPayloadData = localStorage.getItem(
-  'SessionData'
-)
-  ? { SessionData: JSON.parse(localStorage.getItem('SessionData') || '') }
-  : { SessionData: undefined };
+const SessionData: any = Cookies.get('SessionData')
+  ? { SessionData: JSON.parse(Cookies.get('SessionData') || '') }
```

Comment:
You cannot parse if the return value is empty string

Reply:
```
const SessionData: AuthenticationPayloadData =
  Cookies.get('SessionData') !== ''
    ? { SessionData: JSON.parse(Cookies.get('SessionData')) }
    : { SessionData: '' };
```
Causes typescript show errors because Cookies.get("SessionData") may be undefined